README.rst

```diff
@@ -28,3 +28,8 @@ other forms::
 .. |build_status| image:: https://secure.travis-ci.org/rhattersley/pyepsg.png
    :alt: Build Status
    :target: https://travis-ci.org/rhattersley/pyepsg
+
+### Tests
+
+* In order to run unit tests, run `docker build .`. 
+* No specific Python dependencies are required to be installed on your machine.
```

Comment:
I'm not convinced that introducing Docker is appropriate. (Even if it were, it's not really related to the caching change in this PR, so it would be better in a separate PR. That way we can get the caching merged and discuss the use of Docker afterwards.)

Also, if we're not adding any tests then this (and the `Dockerfile`) is no longer needed.